### PR TITLE
Bootstrap Components to Material-UI (login, Signup)

### PR DESCRIPTION
**Pull request by chiragmalik2612**
## Before ( Bootstrap components) 

Screenshot (2248)
Screenshot (2250)

## After ( Material UI components)

Screenshot (2249)

Screenshot (2252)

### CODE REVIEW

File: frontend/src/components/Signup.jsx [hunk 1]
```diff
@@ -107,88 +99,151 @@ function Signup() {
...
-                  type={'text'}
-                  placeholder={'Enter Your Name'}
-                  onChange={(e) => setName(e.target.value)}
+                </Typography>
+                <TextField
+                  fullWidth
+                  size="small"
+                  type=""
+                  placeholder="Enter Your Name"
+                  onChange={(e) => setEmail(e.target.value)}
```
Comment (DhananjayThomble) [inline frontend/src/components/Signup.jsx:135]: can you please test the signup component? I think you've called the wrong function to set the name input field.

**Review by DhananjayThomble — CHANGES_REQUESTED** [review]

File: frontend/src/components/Signup.jsx (hunk 1 above)
Comment (chiragmalik2612) [reply frontend/src/components/Signup.jsx]: > can you please test the signup component? I think you've called the wrong function to set the name input field.

typo fixed , I think everything is fine now

**Comment by chiragmalik2612** [comment]
@DhananjayThomble , will this be count in hacktoberfest?

**Comment by DhananjayThomble** [comment]
> @DhananjayThomble , will this be count in hacktoberfest?

Not now. Let the event begin first.

**Pull request merged**